chemprop/train/make_predictions.py

```diff
@@ -275,17 +299,42 @@ def predict_and_save(
                 for column, smiles in zip(smiles_columns, datapoint.smiles):
                     datapoint.row[column] = smiles
 
+            print_task_names = task_names
```

Comment:
Why is it necessary to add print_task_names as a new variable? Is it doing the same thing as unc_names?